packages/flutter_availability/lib/src/ui/theme.dart

```diff
@@ -0,0 +1,4 @@
+import "package:flutter/material.dart";
+
+///
+final borderRadius = BorderRadius.circular(5);
```

Comment:
To accomodate for multiple designs, it would be better to have this part of the options